sympy/matrices/tests/test_determinant.py

```diff
@@ -378,6 +242,8 @@ def test_cofactor_and_minors():
     assert e.cofactor_matrix() == cm
     assert e.cofactor_matrix(method="bareiss") == cm
     assert e.cofactor_matrix(method="berkowitz") == cm
+    assert e.cofactor_matrix(method="bird") == cm
+    assert e.cofactor_matrix(method="laplace") == cm
```

Comment:
It is a bit hard to see what has or hasn't changed in the tests. Have any tests been changed? Or only tests added for the new solver?

Reply:
Just added those. I did refactor the `test_determinant` a bit in 80d9cc50133281598456f2f2743f651b4fc36b3a, as it would otherwise just give more repetition.